addons/medical_treatment/functions/fnc_canDigGrave.sqf

```diff
@@ -0,0 +1,21 @@
+#include "script_component.hpp"
+/*
+ * Author: Ruthberg, commy2, esteldunedain, drofseh
+ * Checks if a unit can dig a grave at the position of the patient.
+ *
+ * Arguments:
+ * 0: Medic (not used) <OBJECT>
```

Comment:
```suggestion
 * 0: Medic <OBJECT>
```

Reply:
Sorry for the bunch of one line suggestions, I don't know how to include multiple lines into a suggestion when using the browser.

Reply:
> Sorry for the bunch of one line suggestions, I don't know how to include multiple lines into a suggestion when using the browser.

That's alright, thanks for reviewing!